### PR TITLE
test(notification): Add scenario to test 1000+ services notification

### PR DESCRIPTION
**Pull request by jdelpierre**
Refs: MON-34269

## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [x] 24.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/tests/e2e/features/Cloud-notifications/01-notification-create.feature [hunk 1]
```diff
@@ -23,6 +23,25 @@ Feature: Creating a Notification Rule
...
+    And the user clicks on the "Save" button to confirm
+    Then a success message is displayed and the created Notification Rule is displayed in the listing
+    When changes occur in the configured statuses for the selected <resource_type>
+    And the hard state has been reached
+    And the notification refresh_delay has been reached
+    Then an email is sent to the configured <contact_settings> with the configured format
+    Examples:
+      | contact_settings | resource_type                           |
+      | a single contact | host group                              |
+      | two contacts     | host group and services for these hosts |
```
Comment (kduret) [inline centreon/tests/e2e/features/Cloud-notifications/01-notification-create.feature:47]: `a minimum of 1000 services linked to the selected host group and services for these hosts and two contacts` is not understandable

File: centreon/tests/e2e/features/Cloud-notifications/01-notification-create.feature [hunk 2]
```diff
@@ -23,6 +23,25 @@ Feature: Creating a Notification Rule
...
-      | contact_settings                            | resource_type                           |
-      | a single contact                            | host group                              |
-      | two contacts                                | host group and services for these hosts |
+      | contact_settings | resource_type                           |
+      | a single contact | host group                              |
+      | two contacts     | host group and services for these hosts |
+
+  @ignore
+  Scenario Outline: Creating a large volume Notification Rule for <contact_settings>
+    Given a minimum of 1000 services linked to the selected <resource_type> and <contact_settings>
```
Comment (kduret) [inline centreon/tests/e2e/features/Cloud-notifications/01-notification-create.feature:32]:
```suggestion
    Given a minimum of 1000 services linked to a host group and <contact_settings>
```

File: centreon/tests/e2e/features/Cloud-notifications/01-notification-create.feature [hunk 3]
```diff
@@ -23,6 +23,25 @@ Feature: Creating a Notification Rule
...
+    When the user defines a name for the rule
+    And the user selects a <resource_type> with associated events on which to notify
```
Comment (kduret) [inline centreon/tests/e2e/features/Cloud-notifications/01-notification-create.feature:34]:
```suggestion
    And the user selects a host group with its linked services and with associated events on which to notify
```

File: centreon/tests/e2e/features/Cloud-notifications/01-notification-create.feature (hunk 1 above)
Comment (kduret) [inline centreon/tests/e2e/features/Cloud-notifications/01-notification-create.feature:47]:
```suggestion
      | contact_settings |
      | a single contact |
      | two contacts     |
```

**Review by kduret — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Review by paul-oureib — APPROVED** [review]

**Pull request merged**